crates/svm-sdk/src/lib.rs

```diff
@@ -336,15 +336,14 @@ pub use log::log;
 #[macro_use]
 pub mod ensure;
 
-use svm_sdk_alloc;
-use svm_sdk_macros;
-use svm_sdk_storage;
-use svm_sdk_types;
-
-pub use svm_sdk_macros::app;
-
 pub use svm_abi_decoder::{CallData, DecodeError, ReturnData};
 pub use svm_sdk_alloc::{alloc, Ptr};
+pub use svm_sdk_macros::app;
+
+// in order to use the following `global allocator` one should
+// call `extern crate svm_sdk;` (instead of `use svm_sdk;`)
+#[global_allocator]
+pub static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
```

Comment:
It's not clear to me how this should be used. The target exported func should be `svm_alloc`, no? If so, is `ALLOC` being utilized for Rust's allocs?

Reply:
the usage of `#[global_allocator]` implies that `ALLOC` will be the allocator used.
So it means that `svm_alloc` will use it when it's executing.